### PR TITLE
Run2-alca81 Modify and reallocate the scripts used for calibration

### PR DESCRIPTION
**Pull request by bsunanda**
Some of the scripts need modification in view of recent changes in IsoTrack calibration procedure

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@arunhep Can you approve this?

**Comment by arunhep** [comment]
+1

File: Calibration/HcalCalibAlgos/macros/CalibFitPlots.C [hunk 1]
```diff
@@ -190,7 +197,40 @@ std::pair<double,double> fitTwoGauss (TH1D* hist) {
...
+  return std::pair<double,double>(value,error);
+}
+
+std::pair<double,double> fitOneGauss (TH1D* hist, bool debug) {
+  double mean     = hist->GetMean();
+  double rms      = hist->GetRMS();
+  double LowEdge  = mean - 1.5*rms;
+  double HighEdge = mean + 2.0*rms;
+  if (LowEdge < 0.15) LowEdge = 0.15;
+  char option[20];
```
Comment (davidlange6) [inline Calibration/HcalCalibAlgos/macros/CalibFitPlots.C]: hi @bsunanda - please use a std::string here.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@smuzaffar @davidlange6 I believe the IB against which it is tested is wrong. Kindly take care of this. It cannot be caused by this PR

**Comment by slava77** [comment]
@bsunanda 
you can try to rerun because is error is somewhat transient.
Otherwise, wait for the next IB (-2300, if goes as usually scheduled)

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by smuzaffar** [comment]
please test

**Comment by arunhep** [comment]
+1

**Pull request merged**